### PR TITLE
2088

### PR DESCRIPTION
**Pull request by Jkishan544**
Targets #2088 

### Checklist:

- [ ] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/views/hr/menu.blade.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
     @if ((request()->routeIs('applications.job.index')) or (request()->routeIs('applications.internship.index')))
         
         <li class="nav-item c-pointer" data-toggle="modal" data-target="#countIcon">
-            <a class="nav-item nav-link"><i class="fa fa-bitbucket" aria-hidden="true"></i>&nbsp;Total Count</a>
+            <a class="nav-item nav-link"style="color:blue;"><i class="fa fa-bitbucket" aria-hidden="true"></i>&nbsp;Total Count</a>
```
Comment (P4NK4J) [inline resources/views/hr/menu.blade.php:21]: Let's remove the inline style from here and use bootstrap classes.

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left some comments, please check

File: resources/views/hr/menu.blade.php (hunk 1 above)
Comment (Jkishan544) [reply resources/views/hr/menu.blade.php]: please review

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Pull request merged**